azure-pipelines.yml

```diff
@@ -30,6 +30,10 @@ variables:
   YARN_CACHE_FOLDER: $(Pipeline.Workspace)/.yarn
   BLOB_CONTAINER_NAME: '$web'
   REACT_APP_API_URL: 'https://api.io.italia.it'
+  ADB2C_TENANT_AUDIENCE: 'https://iobackoffice.b2clogin.com/iobackoffice.onmicrosoft.com/c2c9dbf8-9fc3-4f69-b8a6-c87d10d0ab06'
+  ADB2C_TENANT_AUTHORITY: 'https://iobackoffice.b2clogin.com/iobackoffice.onmicrosoft.com/B2C_1_backoffice'
+  ADB2C_TENANT_B2CSCOPES: 'https://iobackoffice.onmicrosoft.com/c2c9dbf8-9fc3-4f69-b8a6-c87d10d0ab06/ProfileRead'
+  ADB2C_TENANT_CLIENTID: 'c2c9dbf8-9fc3-4f69-b8a6-c87d10d0ab06'
```

Comment:
can we keep these ones as pipelines variables ? in order to avoid to commit them

Reply:
these changes are unrelated btw and must go in another pr

Reply:
These changes are yet in another PR (rebase when I made this one)
https://github.com/pagopa/backoffice.io.italia.it/pull/28 (waiting for approval)